# StateData
# Data necessary to communicate the current/desired state of the drone
Header header

# Linear Position (m)
float64 x
float64 y
float64 z

# Angular Position (rads)
float64 roll
float64 pitch
float64 yaw

# Linear Velocities (m/s)
float64 vx
float64 vy
float64 vz

# Linear Accelerations (m/s/s)
float64 ax
float64 ay
float64 az
